assets/src/components/AssignmentTable.js

```diff
@@ -233,6 +233,7 @@ function AssignmentTable (props) {
                       >
                         <ProgressBarV2
                           score={a.currentUserSubmission ? a.currentUserSubmission.score : 0}
+                          submitted = {a.currentUserSubmission? a.currentUserSubmission.submittedAt:''}
```

Comment:
The Progress bar is used for both top level bar and assignment level so I am not seeing any conflict there since if the submitted attribute is passed in that it will take effect.

The Passing submitted grades info the top level bar doesn't have an impact since it doesn't overall grades not assignment level grading.

Reply:
@pushyamig I do not quite understand the suggestion here. The submitted but not graded assignment won't be highlighted in color if I remove this line:

submitted = {a.currentUserSubmissiona.currentUserSubmission.submittedAt:''

Reply:
Those are just my thoughts Zhen. No changes required. I will resolve this conversation now